TFT/src/User/API/FanControl.c

```diff
@@ -28,40 +31,49 @@ void fanBuildList(void)
   char* fanCmdTemp[MAX_FAN_COUNT] = FAN_CMD;
   uint8_t fanTypeTemp[MAX_FAN_COUNT] = FAN_TYPE;
 
-  for (uint8_t i = 0, j=0; i < MAX_FAN_COUNT; i++,j++)
+  for (uint8_t i = 0, j = 0; i < MAX_FAN_COUNT; i++, j++)
   {
-    if( infoSettings.fan_count == i ) {
-      if( infoSettings.fan_ctrl_count > 0 ) i= (MAX_FAN_COUNT - MAX_FAN_CTRL_COUNT);
-      else i= MAX_FAN_COUNT;
+    if (infoSettings.fan_count == i)
+    {
+      if (infoSettings.fan_ctrl_count > 0)
+      {
+        i = (MAX_FAN_COUNT - MAX_FAN_CTRL_COUNT);
+      }
+      else
+      {
+        i= MAX_FAN_COUNT;
+      }
     }
-    if(i < MAX_FAN_COUNT) {
+    if (i < MAX_FAN_COUNT)
+    {
       memcpy(fanID+j, fanIDTemp+i, sizeof(fanIDTemp[i]));
       memcpy(fanCmd+j, fanCmdTemp+i, sizeof(fanCmdTemp[i]));
       memcpy(fanType+j, fanTypeTemp+i, sizeof(fanTypeTemp[i]));
     }
   }
 }
 
-void fanControlInit(void) {
+void fanControlInit(void)
+{
   fanBuildList();
 
-  fanQueryEnable = 
-    ( infoSettings.fan_ctrl_count > 0 && 
-      (fanGetTypID( infoSettings.fan_count -1,FAN_TYPE_CTRL_I ) || 
-       fanGetTypID( infoSettings.fan_count -1,FAN_TYPE_CTRL_S ) ) 
+  fanQueryEnable =
+    (infoSettings.fan_ctrl_count > 0 &&
+     (fanGetTypID( infoSettings.fan_count -1, FAN_TYPE_CTRL_I ) ||
+      fanGetTypID( infoSettings.fan_count -1, FAN_TYPE_CTRL_S ))
     );
 }
 
-bool fanIsType(uint8_t i, uint8_t type) {
+bool fanIsType(uint8_t i, uint8_t type)
+{
   return (fanType[i] == type);
 }
 
 void fanSetRcvSpeed(uint8_t i, uint8_t speed)  // by Lori
```

Comment:
Please delete "// by  Lori" comments, I left it there by accident, it's meant only for my personal mods

Reply:
ok I will delete it from all files